src/components/EnvelopeList.vue

```diff
@@ -367,6 +448,28 @@ export default {
 			})
 			this.unselectAll()
 		},
+		markSelectionJunk() {
+			this.selectedEnvelopes.forEach((envelope) => {
+				if (!envelope.flags.$junk) {
+					this.$store.dispatch('toggleEnvelopeJunk', {
+						envelope,
+						removeEnvelope: true,
```

Comment:
```suggestion
						removeEnvelope: await this.$store.dispatch('moveEnvelopeToJunk', envelope),
```

Unfortunately, async/await does not work inside forEach: https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of work well with async/await.